src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1851,6 +1851,15 @@ class ongoingTasksStats extends viewModelBase {
                     case "Sql":
                     case "Olap": {
                         const elementWithData = context.rootStats as EtlPerformanceBaseWithCache;
+                        
+                        if (elementWithData.HasTransformErrors) {
+                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Transform errors:<div class="value">Open notification center for details </div></div>`;
+                        }
+                        
+                        if (elementWithData.HasLoadErrors) {
+                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Load errors:<div class="value">Open notification center for details </div></div>`;
```

Comment:
```suggestion
                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Load errors:<div class="value">Open Notification center for details </div></div>`;
```

Reply:
done, unified all places to `Notification Center`